src/aphroditeInterfaceFactory.js

```diff
@@ -0,0 +1,81 @@
+import { flushToStyleTag } from 'aphrodite/lib/inject';
```

Comment:
The test also needs to be refactored to match, and new tests should be written for the new files to make sure that they are exercised in the test suite.